src/etc/inc/services.inc

```diff
@@ -2204,6 +2204,50 @@ function services_unbound_configure($restart_dhcp = true) {
 			}
 		}
 
+		/* DNS Resolver python integration */
+		$base_folder = '/usr/local';
+		foreach (array('/bin', '/lib') as $dir) {
+			$validate = exec("/sbin/mount | /usr/bin/grep '{$g['unbound_chroot_path']}{$base_folder}{$dir}' 2>&1");
+			if (isset($config['unbound']['python']) && !empty($config['unbound']['python_script'])) {
+
+				// Add DNS Resolver python integration
+				if (empty($validate)) {
+					if (!is_dir("{$g['unbound_chroot_path']}{$base_folder}{$dir}")) {
+						exec("/bin/mkdir -p {$g['unbound_chroot_path']}{$base_folder}{$dir} 2>&1");
+					}
+					exec("/sbin/mount_nullfs -o ro /usr/local{$dir} {$g['unbound_chroot_path']}{$base_folder}{$dir} 2>&1", $output, $retval);
+
+					// Disable Unbound python on mount failure
+					if ($retval != 0) {
+						$config['unbound']['python'] = '';
+						$log_msg = "[Unbound-pymod]: Disabling Unbound python due to failed mount";
+						write_config($log_msg);
+						log_error($log_msg);
+					}
+				}
+			}
+
+			// Remove DNS Resolver python integration
+			elseif (!empty($validate)) {
+				exec("/sbin/umount -t nullfs {$g['unbound_chroot_path']}{$base_folder}{$dir} 2>&1", $output, $retval);
+				if ($retval == 0) {
+					foreach (array( "/usr/local{$dir}", '/usr/local', '/usr') as $folder) {
+						if (is_dir("{$g['unbound_chroot_path']}{$folder}")) {
```

Comment:
Please check that `$g['unbound_chroot_path']` has a sane value before taking a destructive action here. Even though it would fail since the directory wouldn't be empty, it's still better to be safe.

Reply:
At a minimum, preferably also not `/`.

Reply:
I'd like to hardcode it myself... but that is following the other code in Unbound which uses the $g variable setting.